package.json

```diff
@@ -21,7 +21,7 @@
   "devDependencies": {
     "@commitlint/config-conventional": "^12.1.4",
     "@oclif/plugin-help": "^5.1.7",
-    "@oclif/test": "^2.0.2",
+    "@oclif/test": "^1.2.8",
```

Comment:
Why downgrade this version?

Reply:
with 2.0.2 the unit tests are failing. Mike and I discussed and chose to use v1* for the mean time and log a bug against oclif/test to see what was happening.